src/sidebar/components/annotation-omega.js

```diff
@@ -24,12 +25,23 @@ function AnnotationOmega({
   showDocumentInfo,
 }) {
   const createDraft = useStore(store => store.createDraft);
-  const setDefault = useStore(store => store.setDefault);
 
   // An annotation will have a draft if it is being edited
   const draft = useStore(store => store.getDraft(annotation));
   const group = useStore(store => store.getGroup(annotation.group));
 
+  useEffect(() => {
+    // TEMPORARY. Create a new draft for new (non-highlight) annotations
+    // to put the component in "edit mode."
+    if (!draft && isNew(annotation) && !isHighlight(annotation)) {
+      createDraft(annotation, {
+        tags: annotation.tags,
+        text: annotation.text,
+        isPrivate: !isShared(annotation.permissions),
+      });
+    }
+  });
```

Comment:
You need to add the `draft` and `annotation` as dependencies here via the second argument to `useEffect`, otherwise this effect will run on unnecessarily on every render.